demo/halcyon-api.json

```diff
@@ -0,0 +1,2532 @@
+{
+  "info": {
```

Comment:
this file can change in future like adding / modify better we fetch it from json (more can discuss with @mkdbns)